config/routes.rb

```diff
@@ -7,9 +7,8 @@
   mount GovukPublishingComponents::Engine, at: "/component-guide" if defined?(GovukPublishingComponents)
 
   get "/learn-to-drive-a-car", to: 'tasklist#show'
-  get "/get-a-divorce", to: 'tasklist#show_get_a_divorce'
-
-  get "/end-a-civil-partnership", to: 'tasklist#show_end_a_civil_partnership'
+  get "/get-a-divorce", to: 'tasklist#show'
```

Comment:
I think we can even do `get /:base_path, , to: 'tasklist#show, constraints: TasklistConstraints.new`

I'm just trying to think of a way of not having all those base paths written in this app... but then it will have to be inside the Constraint... so not winning much with this approach either... any other ideas?

Reply:
We also need to have published things using the updated schema before we can do funky routing in this way